ArduPlane/Parameters.h

```diff
@@ -93,14 +93,14 @@ class Parameters {
         k_param_scheduler,
         k_param_relay,
         k_param_takeoff_throttle_delay,
-        k_param_skip_gyro_cal, // unused
+        k_param_takeoff_rudder_mix,
```

Comment:
@bugobliterator Because even though the param is unused now it may have been around and had a not default value somewhere in the lifetime of the users autopilot board, and there's no way for us to know if thats true or not. But if it ever is then you would see a totally unexpected value loaded in.

I guess technically we could reclaim any G1 slots that were already unused before PX4 support was introduced... It's probably a huge hassle to go figure out which of those their are... @tridge was this ever done by any chance? I'm not even sure what the ballpark number is on how many params fall into this boat.